forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,391 @@
+        let NodeStatusInfo { health, .. } = self;
+
+        let use_color = color.coloring_enabled();
+        let uptime = (Utc::now() - self.start_time)
+            .to_std()
+            .expect("failed converting to std duration");
+        let fmt_uptime = fmt_duration(uptime);
+        let uptime = format!(
+            "{fmt_uptime} (Started at: {})",
+            self.start_time.with_timezone(&chrono::offset::Local)
+        )
+        .normal();
+
+        let chain_status = self.chain_status().blue();
+        let network = self.network.green();
+        let wallet_address = self
+            .default_wallet_address
+            .clone()
+            .unwrap_or("-".to_string())
+            .bold();
+        let health = {
+            let s = format!("{health:.2}%\n\n");
+            if *health > 85. {
+                s.green()
+            } else if *health > 50. {
+                s.yellow()
+            } else {
+                s.red()
+            }
+        };
+
+        if !use_color {
+            NodeInfoOutput {
+                chain_status: chain_status.clear(),
+                network: network.clear(),
+                uptime: uptime.clear(),
+                health: health.clear(),
+                wallet_address: wallet_address.clear(),
+            }
+        } else {
+            NodeInfoOutput {
+                chain_status,
+                network,
+                uptime,
+                health,
+                wallet_address,
+            }
+        }
+    }
+
+    fn chain_status(&self) -> String {
+        let base_fee =
+            format_balance_string(self.base_fee.clone(), FormattingMode::NotExactNotFixed)
+                .unwrap_or("OutOfBounds".to_string());
+        let behind = format!("{}", humantime::format_duration(self.behind));
+        format!(
+            "[sync: {}! ({} behind)] [basefee: {base_fee}] [epoch: {}]",
+            self.sync_status, behind, self.epoch
+        )
+    }
+}
+
+fn fmt_duration(duration: Duration) -> String {
+    let duration = format_duration(duration);
+    let duration = duration.to_string();
+    let duration = duration.split(' ');
+    let format_duration = duration
+        .filter(|s| !s.ends_with("us"))
+        .filter(|s| !s.ends_with("ns"))
+        .filter(|s| !s.ends_with("ms"))
+        .map(|s| s.to_string());
+    let format_duration: Vec<String> = format_duration.collect();
+    format_duration.join(" ")
+}
+
+struct NodeInfoOutput {
+    chain_status: ColoredString,
+    network: ColoredString,
+    uptime: ColoredString,
+    health: ColoredString,
+    wallet_address: ColoredString,
+}
+
+impl InfoCommand {
+    pub async fn run(&self, config: Config, opts: &CliOpts) -> anyhow::Result<()> {
+        // uptime
+        let start_time = start_time(&config.client.rpc_token)
+            .await
+            .map_err(handle_rpc_err)?;
+
+        let network = chain_get_name((), &config.client.rpc_token)
```

Comment:
Our endpoint `ChainGetName` could be replaced by the one from the Filecoin api `Filecoin.StateGetNetworkParams`. This could be done in another PR though.